dwave/system/samplers/clique.py

```diff
@@ -78,20 +79,31 @@ def sample(self, bqm, **parameters):
         Returns:
             :obj:`dimod.SampleSet`
         """
-        if 'per_qubit_coupling_range' in self.child.properties.keys():
+        if any(('per_qubit_coupling_range' in self.child.properties.keys(),
+                'per_group_coupling_range' in self.child.properties.keys())):
 
-            min_lim = self.child.properties['per_qubit_coupling_range'][0]
-            max_lim = self.child.properties['per_qubit_coupling_range'][1]
+            try:
```

Comment:
Just curious why this is structured as a `try:`/`except` rather than an `if per_qubit_coupling_range in self.child.properties.keys():  ... elsif per_group_coupling_range in self.child.properties.keys())`

Reply:
No particular reason.  I suppose `if..else` is more explicit so that would probably be better.

Reply:
Made that change.